app/src/main/java/chat/rocket/android/service/RocketChatWebSocketThread.java

```diff
@@ -52,7 +51,6 @@
       NewMessageObserver.class,
       CurrentUserObserver.class,
       ReactiveNotificationManager.class,
-      NotificationItemObserver.class,
```

Comment:
Some of the guys were getting upset by this notification. I'll try a build without it and get some feedback.